doc/users/python_tuto.rst

```diff
@@ -0,0 +1,83 @@
+Python Client Tutorial
```

Comment:
Minor: Why is this `tuto` rather than `python_tutorial.rst`?

Reply:
Some French-ism for you. Renamed to `python_tutorial.rst` 👍